### PR TITLE
Jsshuffle

### PR DESCRIPTION
**Pull request by 0xmtn**

### CODE REVIEW

**Comment by 0xmtn** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -18,6 +18,76 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+a=[1,2,3,4,5,6,7,8];
+b = shuffle(a);
+console.log(b);
+//[2, 7, 8, 6, 5, 3, 1, 4]
+```
+
+Also, it can be used as a function in Array.prototype like this:
+
+```javascript
+Array.prototype.shuffle = function(){
```
Comment (loverajoel) [inline README.md]: this is a bad practice

File: README.md [hunk 2]
```diff
@@ -18,6 +18,50 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+a=[1,2,3,4,5,6,7,8];
+b = shuffle(a);
+console.log(b);
+//[2, 7, 8, 6, 5, 3, 1, 4]
+```
+
+### Pure JS version by [@mvedie](https://github.com/mvedie)
+
+```javascript
+Array.prototype.shuffle = function() {
```
Comment (loverajoel) [inline README.md]: remove the prototype, it's a bad practice

File: README.md [hunk 3]
```diff
@@ -18,6 +18,50 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+function shuffle() {
+    return this.slice(0).sort(function() {
+        return 0.5 - Math.random();
+    });
+};
+```
+
+An Example:
+```javascript
+my_array = [1,2,3,3,4,5,6]
```
Comment (loverajoel) [inline README.md]: `var myArray`

File: README.md [hunk 4]
```diff
@@ -18,6 +18,50 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+        arr[i] = arr[j];
+        arr[j] = temp;
+    }
+    return arr;    
+}
+```
+An example:
+
+```javascript
+a=[1,2,3,4,5,6,7,8];
```
Comment (loverajoel) [inline README.md]: `var a = [1, 2, 3, 4, 5, 6, 7, 8];`

File: README.md [hunk 5]
```diff
@@ -18,6 +18,50 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
...
+b = shuffle(a);
```
Comment (loverajoel) [inline README.md]: `var b`

**Comment by loverajoel** [comment]
Thanks @0xmtn !

**Comment by 0xmtn** [comment]
my pleasure :thumbsup:

**Pull request merged**